packages/backend/src/open_payments/payment/incoming/service.ts

```diff
@@ -372,7 +376,9 @@ async function handleDeactivated(
       tenantId: incomingPayment.tenantId,
       webhooks: finalizeWebhookRecipients(
         [incomingPayment.tenantId],
-        deps.config
+        deps.config,
+        incomingPayment.metadata,
```

Comment:
Now that we are not using `metadata` for card info, we can instead pass in `initiatedBy`